libs/community/langchain_community/vectorstores/milvus.py

```diff
@@ -124,6 +126,7 @@ def __init__(
         search_params: Optional[dict] = None,
         drop_old: Optional[bool] = False,
         auto_id: bool = False,
+        enable_dynamic_field: bool = False,
```

Comment:
Should we move this after the *, in case users are passing in arguments positionally?